### DISCUSSION TITLE
Is there any way to change/order ciphers used for connection?

### DISCUSSION THREAD

**Discussion by oczkers** [encode/httpx#3299] [General] [answered]
Hello, i know it might not be the place to discuss it but i've got problem with cloudflare detecting tls fingerprint and blocking my requests. Could You please help me to find way for setting cipher suites in order or at least spoof cipher suite and tls version?

related info
https://www.zenrows.com/blog/what-is-tls-fingerprint

**Comment by lovelydinosaur** [comment, marked as answer]
For documentation on dealing with `SSLContext` you can refer to...

* https://www.python-httpx.org/advanced/ssl/
* https://docs.python.org/3/library/ssl.html

I can't help with your particular request, but worth noting that you can pass a custom `SSLContext` instance with `verify=<custom SSL context>`.
